client/src/components/Carousel.jsx

```diff
@@ -15,6 +15,7 @@ class Carousel extends React.Component {
   getPhotos(id) {
     axios.get(`/api/restaurants/${id}/photos`)
       .then((response) => {
+        console.log("client", response)
```

Comment:
Handle 400 error here. If response.length === 0, you know the db didnt fail, the information just doesnt exist aka bad request from client.